src/control/nav2_controller_example/include/nav2_controller_example/dwa_controller.hpp

```diff
@@ -17,17 +17,21 @@
 
 #include <string>
 #include "nav2_tasks/follow_path_task.hpp"
+#include "geometry_msgs/msg/twist.hpp"
 
 namespace nav2_controller_example
 {
 
 class DwaController : public nav2_tasks::FollowPathTaskServer
 {
 public:
+  typedef geometry_msgs::msg::Twist CmdVel;
```

Comment:
Should we "standardize" on using instead of typedef? That seems to be the style with ROS2.

Reply:
Done